### DISCUSSION TITLE
Nokia MikroMikko 1

### DISCUSSION THREAD

**Discussion by nikoniem** [mamedev/discussions#72] [Ideas]
Hi!

I can provide BIOS dumps for MikroMikko 1 models: MM1G M4 (2x320kB) and MM1 M7 (1x640kB + 5MB Winchester).
Is Curt Coder or someone else interested?

-Niko

**Comment by rb6502** [comment]
I don't know if Curt is active right now, but we're always interested in BIOS dumps.  If you can use the Contact page to point us to them (posting links to ROMs on Github is questionable) we'd appreciate it!  https://www.mamedev.org/contact.html

If that doesn't work for you for some reason, let me know.

**Comment by curt-coder** [comment]
Yes, I'm interested. Please send the dumps over and I'll add them.
